### PR TITLE
feat: update setup.py, removal of urllib3 module version upper boundary requirement [APE-1610]

### PR DESCRIPTION
**Pull request by popraf**
Update setup.py - removal of urllib3 module version requirement

### What I did

fixes: #1741

### How I did it

Updated `setup.py` by removal of `urllib3` module version upper boundary requirement
### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by fubuloubu — APPROVED** [review]

**Comment by fubuloubu** [comment]
Can you give a quick explanation why the library pin is troublesome?

**Comment by Aviksaikat** [comment]
> Can you give a quick explanation why the library pin is troublesome?

Hey, thanks for looking into this. As ape is not using urllib heavily I think it's safe to remove the upper bound as it creates conflicts with other libraries that depend on specific versions of urllib.

**Comment by antazoey** [comment]
We could also bump it completely to the 2.0 range, as we are about to do big breaking change release

File: setup.py [hunk 1]
```diff
@@ -112,7 +112,7 @@
         "SQLAlchemy>=1.4.35",
         "tqdm>=4.62.3,<5.0",
         "traitlets>=5.3.0",
-        "urllib3>=1.26.16,<2",
+        "urllib3>=1.26.16",
```
Comment (antazoey) [inline setup.py:115]:
```suggestion
        "urllib3>=2.1.0,<3",
```

**Review by antazoey — COMMENTED** [review]
bump up instead?

File: setup.py (hunk 1 above)
Comment (antazoey) [reply setup.py:115]: i bumped here instead: https://github.com/ApeWorX/ape/pull/1773

**Pull request merged**